src/app/edit-profile/edit-profile.component.html

```diff
@@ -0,0 +1,16 @@
+
+<div class="container">
+
+  Name: <input type="text" [(ngModel)]="name">{{name}}<br>
+  Username: <input type="text" [(ngModel)]="username">{{username}}<br>
+  Photo Link: <input type="text" [(ngModel)]="photo">{{photo}}<br>
+  Bio: <input type="text" [(ngModel)]="bio">{{bio}}<br>
+  Email: <input type="text" [(ngModel)]="email">{{email}}<br>
+
+  <button (click)="update(name, username, photo, bio, email)"
+          [routerLink]="['../user-profile']"
+          class="btn-danger btn-xs">
```

Comment:
Are this classes defined by you or them came with bootstrap?

Reply:
these are bootstrap classes